### ISSUE TITLE
Move from Google Maps API to ESRI account

### ISSUE THREAD

**Issue by jmartini** [codeforboston/voiceapp311#170]
The City of Boston already has an account with ESRI (https://www.esri.com) and would prefer to use that instead of signing up for a new Google Maps API. Looking at their documentation, we should be able to get all the same info we need from the ESRI API.

**Comment by GreenGiraffe1** [comment]
ERSI develops the ArcGIS platform, and I believe this is the API we want to hit:
[ArcGIS Route Service](https://developers.arcgis.com/rest/network/api-reference/route-synchronous-service.htm)

This link (from the above page) seems especially useful [Get a route and directions](https://developers.arcgis.com/labs/rest/get-a-route-and-directions/).

**Comment by chrislreich** [comment]
The Google Maps API functionality is contained within mycity/utilities/google_maps_utils.py

The only place that uses this module is another utilities module, mycity/utilities/finder/.

The finder utility makes use of the two features of the Google Maps API: driving distance and driving time from origin to destination(s). It appears that these two features can be replicated in the ESRI/ArcGIS API:

https://developers.arcgis.com/labs/rest/get-a-route-and-directions/

For completeness' sake, the only intent that currently (that I can find) uses the Finder module is the snow_parking_intent, which informs the user of the how far and how long the drive is to the closest snow emergency parking.

In summary, it should be possible to replace the Google Maps API with ESRI/ArcGIS. Considering we currently only use two features of the Google Maps API, we should be able to migrate with without significant effort. The next step would be to obtain ESRI credentials from the City of Boston and start the migration.

**Comment by jmartini** [comment]
It looks like this lab might be useful as well: https://developers.arcgis.com/labs/rest/route-to-the-closest-destination/

Documentation can be found here: https://developers.arcgis.com/rest/network/api-reference/closest-facility-synchronous-service.htm

**Comment by jmartini** [comment]
Closed with #176
